src/main/java/org/gridsuite/study/server/service/RemoteServices.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.service;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.NullNode;
+import lombok.NonNull;
+import org.gridsuite.study.server.RemoteServicesProperties;
+import org.gridsuite.study.server.dto.ServiceStatusInfos;
+import org.gridsuite.study.server.dto.ServiceStatusInfos.ServiceStatus;
+import org.gridsuite.study.server.service.client.RemoteServiceName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.info.InfoEndpoint;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@Service
+public class RemoteServices {
```

Comment:
I know it's ugly but maybe rename into RemoveServicesService ?

Reply:
ugly yes... dunno

Reply:
After discutions with team lead, let's rename this to RemoteServicesMonitor instead.